lib/widgets/compose_box.dart

```diff
@@ -22,6 +22,63 @@ import 'store.dart';
 import 'text.dart';
 import 'theme.dart';
 
+/// Compose-box styles that differ between light and dark theme.
+///
+/// These styles will animate on theme changes (with help from [lerp]).
+class ComposeBoxTheme extends ThemeExtension<ComposeBoxTheme> {
+  factory ComposeBoxTheme.light(BuildContext context) {
+    return ComposeBoxTheme._(
+      boxShadow: null,
+    );
+  }
+
+  factory ComposeBoxTheme.dark(BuildContext context) {
+    return ComposeBoxTheme._(
+      boxShadow: [BoxShadow(
+        color: DesignVariables.dark.bgTopBar,
+        offset: const Offset(0, -4),
+        blurRadius: 16,
+        spreadRadius: 0,
+      )],
+    );
+  }
```

Comment:
Ah a small improvement on my proposal above 🙂 these don't actually need a `BuildContext` so they can just be static final fields:

```dart
  static final light = ComposeBoxTheme._(
    boxShadow: null,
  );

  static final dark = ComposeBoxTheme._(
    boxShadow: [BoxShadow(
      color: DesignVariables.dark.bgTopBar,
      offset: const Offset(0, -4),
      blurRadius: 16,
      spreadRadius: 0,
    )],
  );
```